### PR TITLE
feat: create command helpers

### PR DESCRIPTION
**Pull request by karitham**
also move _example to 0_example since gopls doesn't like the _example directory name.

### CODE REVIEW

**Comment by jolheiser** [comment]
Meh, screwed up my branch. Check out https://github.com/Karitham/corde/commit/6d6969317c4cc9b3d487469ad0cce9b8537d6680

**Comment by jolheiser** [comment]
As mentioned in discord, I think generation might be best for this, if at all.

Generics attempt was pain.

**Comment by karitham** [comment]
Seems like the case, yeah,

My efforts are currently focused around #7, but will come back to that once I'm done.

Open to any other suggestions, truthfully, the generics part was a good idea from a user POV, but it's an ugly and non-extensible hack.

File: internal/cmd/gen-register/main.go [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+		Description: description,
+		Required:    required,
+	}
+
+	for _, choice := range choices {
+		o.Choices = append(
+			o.Choices,
+			Choice[any]{Name: choice.Name, Value: choice.Value},
+		)
+	}
```
Comment (jolheiser) [inline internal/cmd/gen-register/main.go:39]: Glad you kept at least one of my hacks. 😌

**Review by jolheiser — APPROVED** [review]

**Pull request merged**